.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
```

Comment:
I'm wondering if updating highlightJs on every commit may be a little too much. It will also slow down the development workflow (not that much is happening these days) if we have to wait for a network request to complete before every commit. 

What do you think about making this manual? 

A couple of questions to help us answer:
1. How often does this library get updated?
2. How significant are the updates?
3. Is it crucial that it always remain up to date?
4. If this just blindly fetches the latest version of highlight, could we introduce a breaking change unwittingly when a new major version is released?

Reply:
I think you make a good point here, and I agree with you. We can update it as we test libraries. The Apex library will be updated in a few weeks, so we can make an update here after that.